models/stg_recurly__account_balance_history.sql

```diff
@@ -23,7 +23,8 @@ final as (
         cast(account_updated_at as {{ dbt_utils.type_timestamp() }}) as account_updated_at,
         cast(amount as {{ dbt_utils.type_float() }}) as amount,
         currency,
-        past_due
+        past_due,
+        row_number() over (partition by account_id order by account_updated_at desc) = 1 as is_most_recent_record
```

Comment:
I liked how you added the is_most_recent_record to the staging models we use downstream. I felt it would be relevant to add to the other history models as well. Even though we may not use them downstream.

Reply:
Same comment and note for all the other staging model updates.

Reply:
Works for me!